programs/bpf_loader/src/lib.rs

```diff
@@ -1398,9 +1398,19 @@ fn execute<'a, 'b: 'a>(
         };
         create_vm_time.stop();
 
-        execute_time = Measure::start("execute");
         let (compute_units_consumed, result) = vm.execute_program(executable, !use_jit);
         drop(vm);
+        invoke_context.execute_time.stop();
+        saturating_add_assign!(
+            invoke_context.timings.execute_us,
+            invoke_context.execute_time.as_us()
+        );
+        invoke_context.timings.execute_us = invoke_context
+            .timings
+            .execute_us
+            .saturating_add(invoke_context.execute_time.as_us())
```

Comment:
You're adding `invoke_context.execute_time.as_us()` twice here (also line 1404-1407).

Reply:
You are absolutely right, talking about over-counting